reconcile/utils/oc.py

```diff
@@ -711,27 +711,33 @@ def sa_get_token(self, namespace, name):
         cmd = ["sa", "-n", namespace, "get-token", name]
         return self._run(cmd)
 
+    def _process_api_resources(self, results):
+        for line in results:
+            r = line.split()
+            kind = r[-1]
+            namespaced = r[-2].lower() == "true"
+            group_version = r[-3].split("/", 1)
+            # Core group (v1)
+            group = ""
+            api_version = group_version
+            if len(group_version) > 1:
+                # group/version
+                group = group_version[0]
+                api_version = group_version[1]
+            obj = OCDeprecatedApiResource(kind, group, api_version, namespaced)
+            d = self.api_resources.setdefault(kind, [])
+            d.append(obj)
+
     def get_api_resources(self):
         with self.api_resources_lock:
             if not self.api_resources:
                 cmd = ["api-resources", "--no-headers"]
-                results = self._run(cmd).decode("utf-8").split("\n")
```

Comment:
@cubismod, I wonder if the way of `self._run(cmd).encode().decode("utf-8").split("\n")` would work here?

Example:

```python
>>> s = "Hello"
>>> str(s, "utf-8")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: decoding str is not supported
>>> s.decode("utf-8")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'decode'. Did you mean: 'encode'?
>>> s.encode().decode("utf-8")
'Hello'
```

Reply:
Huh yeah I think that would work too